src/MessagePack/Internal/ReflectionExtensions.cs

```diff
@@ -15,6 +15,11 @@ public static bool IsNullable(this System.Reflection.TypeInfo type)
             return type.IsGenericType && type.GetGenericTypeDefinition() == typeof(System.Nullable<>);
         }
 
+        public static bool IsNullable(this Type type)
```

Comment:
Since the containing type is internal, just change the original method signature to take Type rather than duplicate it to another overload, please.